rotkehlchen/data/globaldb_v2_v3_assets.sql

```diff
@@ -719,6 +719,7 @@ INSERT INTO assets(identifier, type, name) VALUES
 ("eip155:56/erc20:0x1e155e26085Be757780B45a5420D9f16a938f76b", "C", "KaraStar KARA"),
 ("eip155:56/erc20:0x86a53fcd199212FEa44FA7e16EB1f28812be911D", "C", "Inflation Hedging Coin"),
 ("eip155:56/erc20:0x4ffA6BB42d1a1A6d8e79935CcF1457d55deCff3f", "C", "REV3AL");
+
```

Comment:
For simplicity of the code in `v2_v3` globaldb upgrade. After adding these lines it is possible to split statements by `\n\n` and execute them via `.execute` instead of `.executescript`